session_4/blogspot/app/views/articles/edit.html.erb

```diff
@@ -0,0 +1,14 @@
+<br>
+<font color="#3385ff">
```

Comment:
Instead of manually specifying color, you can make use of bootstrap colors: https://getbootstrap.com/docs/4.5/utilities/colors/